### PR TITLE
Editor tour / onboarding feature: styling and animation

### PR DESCRIPTION
**Pull request by kimprice**
This PR introduces the following changes:
- transition animation for teaching bubbles
- larger font size
- border around bubble instead of drop shadow
- removed toolbox from being highlighted in the workspace callout
- made tour in order of anticipated use (toolbox -> workspace -> simulator -> share -> download)
- (Jacqueline) updated content to be more concise
- increased padding/margin for readability
- aligned lower edge of button and progress indicator / step count
- lighter font color for steps

Closes https://github.com/microsoft/pxt-microbit/issues/5009, closes https://github.com/microsoft/pxt-microbit/issues/5013

Try it out here: https://makecode.microbit.org/app/145250d33320d7b8b2a9b197a224a222bdeda9b2-8b591e28cd

**Still need to do some refactoring to clean up the teaching bubble component and possibly move out the arrow and cutout logic into their own components.**

EditorTour

### CODE REVIEW

**Review by jwunderl — APPROVED** [review]
Nice!

File: react-common/components/controls/TeachingBubble.tsx [hunk 1]
```diff
@@ -91,15 +95,20 @@ export const TeachingBubble = (props: TeachingBubbleProps) => {
...
-
+        setPosition(cutoutBounds, bubble, bubbleBounds, bubbleArrow, bubbleArrowOutline);
     }
 
     const getCutoutBounds = (targetBounds: DOMRect, targetElement: HTMLElement): CutoutBounds => {
         let cutoutTop = targetBounds.top;
         let cutoutLeft = targetBounds.left;
         let cutoutWidth = targetBounds.width;
         let cutoutHeight = targetBounds.height;
+        if (targetContent.sansQuery) { // TO DO - take care of cases when sansElement is not to the left of targetElement
```
Comment (srietkerk) [inline react-common/components/controls/TeachingBubble.tsx:106]: What type of content are we not wanting to include in the cutouts?

File: webapp/src/components/onboarding/EditorTour.tsx [hunk 2]
```diff
@@ -4,22 +4,23 @@ import { Location, TeachingBubble, TargetContent } from "../../../../react-commo
...
-    description: lf("The simulator shows what your program will look like running on a micro:bit."),
+    description: lf("See what your code looks like running on a micro:bit!"),
     targetQuery: "#boardview",
     location: Location.Right,
 };
 
 const Toolbox: TargetContent = {
     title: lf("Toolbox"),
-    description: lf("The toolbox is organized into different categories with blocks of code you can drag out and use in your program. You can search for specific blocks, open the extensions gallery for more blocks, or click Advanced to expand the toolbox categories shown."),
+    description: lf("Drag out blocks of code from the Toolbox categories into the Workspace."),
```
Comment (thsparks) [inline webapp/src/components/onboarding/EditorTour.tsx:14]: nit: Something like "Drag blocks of code out of the Toolbox categories into the Workspace" feels a little cleaner to me, but I think it's just a matter of personal preference.

File: webapp/src/components/onboarding/EditorTour.tsx [hunk 3]
```diff
@@ -32,15 +33,15 @@ const Share: TargetContent = {
 
 const Download: TargetContent = {
     title: lf("Download"),
-    description: lf("Download your program to your micro:bit."),
+    description: lf("Download your program to the micro:bit."),
```
Comment (thsparks) [inline webapp/src/components/onboarding/EditorTour.tsx:36]: Another nit:

```suggestion
    description: lf("Download your program onto the micro:bit."),
```

**Review by thsparks — APPROVED** [review]
Some wording nit-picks (feel free to ignore), but looks good overall!

**Pull request merged**